config/karma.config.js

```diff
@@ -3,11 +3,12 @@
 
 import webpackConfig from './webpack.config';
 
-var coverageReporters = [{
+const coverageReporters = [{
   type: 'text-summary',
 }];
 
-var newWebpackConfig = Object.assign({}, webpackConfig, {
+// eslint-disable-next-line prefer-const
+let newWebpackConfig = Object.assign({}, webpackConfig, {
```

Comment:
An example where eslint gets it wrong. I'm modifying this later on in the file. Technically const object props can be modified but semantically using let is more correct.